src/osvr/PluginHost/RegistrationContext.cpp

```diff
@@ -51,9 +51,6 @@
 namespace osvr {
 namespace pluginhost {
     static const auto PLUGIN_HOST_LOGGER_NAME = "PluginHost";
-#ifdef _MSC_VER
```

Comment:
See `src/osvr/PluginHost/RegistrationContext.cpp`, circa line 168 where we check to see if the `.manualload` suffix appears at the end of the name.